### PR TITLE
FIX: Test updates to comply with DataObject::get_one cahnge

### PR DESCRIPTION
**Pull request by sminnee**
See https://github.com/silverstripe/silverstripe-framework/pull/7083

### CODE REVIEW

File: tests/model/SiteTreeTest.php [hunk 1]
```diff
@@ -540,7 +540,7 @@ public function testDeleteFromLiveOperatesRecursivelyStrict()
         $parentPage->doUnpublish();
 
         Versioned::set_stage(Versioned::LIVE);
-        $this->assertFalse(DataObject::get_by_id('Page', $pageAbout->ID));
+        $this->assertNull(DataObject::get_by_id('Page', $pageAbout->ID));
         $this->assertFalse(DataObject::get_by_id('Page', $pageStaff->ID));
```
Comment (tractorcow) [inline tests/model/SiteTreeTest.php]: Don't forget the tests below...

Comment (sminnee) [inline tests/model/SiteTreeTest.php]: :+1:

**Comment by tractorcow** [comment]
Waiting for tests to go green.

**Comment by tractorcow** [comment]
Ok to merge on green; Ignore behat tests since they are failing at the moment due to upstream issues.

**Pull request merged**